lms/envs/devstack.py

```diff
@@ -260,7 +267,10 @@ def should_show_debug_toolbar(request):
     'use-jwt-cookie',
 )
 
-LOGIN_REDIRECT_WHITELIST = [CMS_BASE]
+LOGIN_REDIRECT_WHITELIST = [
+    CMS_BASE,
+    ENTERPRISE_LEARNER_PORTAL_NETLOC,
```

Comment:
while you're in here, do you think you can add the Enterprise Admin Portal? http://localhost:1991

Reply:
might as well!